src/io/multi_writer.cr

```diff
@@ -0,0 +1,52 @@
+module IO
+  # An IO which writes to a number of underlying writer IOs.
+  #
+  # ```
+  # io1 = IO::Memory.new
+  # io2 = IO::Memory.new
+  # writer = IO::MultiWriter.new(io1, io2)
+  # writer.puts "foo bar"
+  # io1.to_s # => "foo bar\n"
+  # io2.to_s # => "foo bar\n"
+  # ```
+  class MultiWriter
+    include IO
+
+    # If `sync_close` is true, closing this IO will close all of the underlying
+    # IOs.
+    property? sync_close
+    getter? closed = false
+
+    @writers : Array(IO)
```

Comment:
I think this can be removed as it will be correctly inferred from the constructor below